transforms:
  T_robot_chess:
    - [-1.0, 0.0, 0.0, 0.358]
    - [0.0, 1.0, 0.0, 0.03]
    - [0.0, 0.0, -1.0, 0.006]
    - [0.0, 0.0, 0.0, 1.0]

  T_chess_cam1:
    - [-0.988, 0.1015, -0.1167, 0.2521]
    - [-0.1544, -0.6932, 0.704, -0.728]
    - [-0.0095, 0.7136, 0.7005, -0.5255]
    - [0.0, 0.0, 0.0, 1.0]
